lib/i18n/default.js

```diff
@@ -110,6 +110,22 @@ function pluralize(noun) {
     return undefined;
 }
 
+function isGoodWord(word) {
+    // filter out words that cannot be in the dataset,
+    // because they would be either tokenized/preprocessed out or
+    // they are unlikely to be used with voice
+    // [Arabic_chars Arabic_Digits Farsi_supplements ...]
+    return undefined;
```

Comment:
Should always return `true` right? Absent any language-specific info, every word is a good word.

Reply:
I mean in python we usually let base class return "NotImplementedError()". Here are real default is American English. I can change it though. Should I change "isGoodSentence" and "isGoodNumber" to true as well?

Reply:
No the default is used for all languages for which we don't have language specific code. I.e. it will be used for Spanish, French, German, Japanese, etc.
So yeah you need a conservative default.